src/lib.rs

```diff
@@ -176,7 +176,17 @@ where
 
     let mut duration = Duration::from_secs(0);
     loop {
-        match reader.read_exact(&mut header_buffer[..]) {
+        //skip over all 0x00 bytes (these are probably incorrectly added padding bytes for id3v2)
+        header_buffer[0] = 0;
+        while header_buffer[0] == 0{
```

Comment:
Formatting nitpicks:
- Missing a space before the curly brace line 181
- Comment should begin with `// Skip` instead of `//skip`